filters.js

```diff
@@ -55,6 +55,7 @@ var filters = {
   last: v => v[v.length - 1],
   lstrip: v => stringify(v).replace(/^\s+/, ""),
   map: (arr, arg) => arr.map(v => v[arg]),
+  sum: (arr, arg) => arr.reduce((acc, v) => acc + (v[arg] ?? v), 0),
```

Comment:
- What if the type is an object without having `arg` property?
- What if the value of the property is `undefined` or `null` or `NaN`?
- Add tests

Reply:
+1 

also you need to handle boxed types (duration/date/currency)

Reply:
How should we handle for duration/currency? should we return the sum of their values only? @codebreach